### DISCUSSION TITLE
CODEOWNERS should support the directory and file with the name of non-English language

### DISCUSSION THREAD

**Discussion by VictorHuu** [community/community#64754] [Repositories] [answered]
### Select Topic Area

Question

### Body

I want to assign the owners of my repo,but it didn't work.

src/components/用户界面/medical.vue
The above line is invalid!

So I consulted to GPT,but he told me to add the note like '# coding: utf-8' ,but it still didn't work.

**And I want to figure out whether CODEOWNERS support non-ASCII naming or not,if yes,what should I do to resolve this?**

**Comment by arafajs** [comment, marked as answer]
GitHub does not allow non-ASCII characters in filenames by default.

To resolve this, you can either:

1. Rename the file to remove the non-ASCII characters.
2. Set the `force-utf8` configuration option to` true` in your repository's `.gitattributes` file. This will tell Git to treat all filenames as UTF-8, even if they contain non-ASCII characters.
